src/eva/transforms/select_time.py

```diff
@@ -0,0 +1,90 @@
+# (C) Copyright 2021-2022 NOAA/NWS/EMC
+#
+# (C) Copyright 2021-2022 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+
+from eva.transforms.transform_utils import parse_for_dict, split_collectiongroupvariable
+from eva.transforms.transform_utils import replace_cgv
+from eva.utilities.config import get
+from eva.utilities.logger import Logger
+
+
+# --------------------------------------------------------------------------------------------------
+#  Select a variable by the Time dimension in form YYYYMMDDHH.  If the input is a single time
+#  then values for the specified variable will be selected for that cycle and added to
+#  data_collections.
+#
+#  If two cycle times are included they will be used as the bounds for a time slice, and the
+#  average of that span will be added to the data_collections.  Note that when specifying a slice
+#  the order must be older (lower) to newer (higher).
+
+def select_time(config, data_collections):
+
+    # Create a logger
+    logger = Logger('SelectTimeTransform')
+
+    # Parse config for dictionary
+    [collections, groups, variables] = parse_for_dict(config, logger)
+
+    # Parse config for the expression and new collection/group/variable naming
+    new_name_template = get(config, logger, 'new name')
+    starting_field_template = get(config, logger, 'starting field')
+
+    # Get cycles from yaml file and confirm there are 1 or 2 times
+    cyc = get(config, logger, 'cycles')
+    if type(cyc) is list:
+        cycles = cyc
+    else:
+        cycles = [cyc]
```

Comment:
`cycles = cyc if type(cyc) is list else [cyc]`

Reply:
Cool.  Done.